src/model/embedding/noms2embeddings_model.py

```
@@ -26,37 +26,54 @@ def _create_and_save_embeddings(
+    job.meta['general_status'] = "Extracting brands"
+    job.save_meta()
+
     noms_names_list = df_noms_with_features['name'].to_list()
     df_noms_with_features['brand'] = ner_service.predict(noms_names_list)
 
+    job.meta['general_status'] = "Building metadatas"
+    job.save_meta()
+
     # Получаем метаданные всех номенклатур с характеристиками
     metadatas = get_noms_metadatas_with_features(df_noms_with_features)
     for i, metadata in enumerate(metadatas):
```

Ruff

Loop control variable `metadata` not used within loop body

Rename unused `metadata` to `_metadata`

(B007)